env/src/main/scala/tofu/env/Env.scala

```diff
@@ -70,7 +70,8 @@ sealed trait Env[E, +A] {
     mapTask3(eb, ec)(Task.parMap3(_, _, _)(f))
 
   def zip[B](t: Env[E, B]): Env[E, (A, B)]                 = map2(t)((a, b) => (a, b))
-  def >>[B](t: Env[E, B]): Env[E, B]                       = map2(t)((_, x) => x)
+  def >>[B](t: => Env[E, B]): Env[E, B]                    = flatMap(_ => t)
+  def *>[B](t: Env[E, B]): Env[E, B]                       = map2(t)((_, x) => x)
```

Comment:
What about `<<` and `<*` and possibly `&>` `<&` for `parMap2` ?

Reply:
`<<` is already there and it seems there is no way to avoid eager evaluation of second argument since it must be evaluated first.
I will add the operators you mentioned and maybe a few more that can be handy (any ideas for those?)

Reply:
`<<` can be made lazy on pseudo-this parameter via extension like
```scala 
implicit class LazyEnvOps[E, A](private self: => Env[E, A])
```

Reply:
Can you please elaborate on that?
I tried something like this and it didn't work:
```scala
implicit class LazyEnvOps[E, A](self: => Env[E, A]) {
  def <<[B](that: Env[E, B]): Env[E, A] = that.flatMap(_ => self)
}

def env1: Env[Unit, Unit] = {
  println("env1 evaluated")
  Env.unit[Unit]
}

def env2: Env[Unit, Unit] = {
  println("env2 evaluated")
  Env.unit[Unit]
}

env1 << env2
```
It prints both statements. Am I missing something?

Reply:
Well you may do something like
```scala
implicit class LazyEnvOps[E, A](self: => Env[E, A]) {
  def <<[B](that: Env[E, B]): Env[E, A] =  Env.defer(self) <* that
}
```

On the other hand, it would be more rational to make it lazy on the second argument 

```scala

def flatTap[B](that: A => Env[E, B]): Env[E, A] = flatMap(x => that as x)
def <<[B](that: => Env[E, B]): Env[E, A] = flatTap(_ => that)
```

Reply:
Sorry I misunderstood designation of `<<` at first.
I added `flatTap` and operators for `parMap2`. Any ideas for more operators?